conf/iptables.conf.example

```diff
@@ -115,17 +123,9 @@
 %%input_inter_isol_vlan_if%%
 
 :input-internal-inline-if - [0:0]
-# DHCP
--A input-internal-inline-if --protocol udp --match udp --dport 67  --jump ACCEPT
--A input-internal-inline-if --protocol tcp --match tcp --dport 67  --jump ACCEPT
 # DNS
-# allow unregistered users and isolated users to reach it for DNAT purposes but prevent registered ones
--A input-internal-inline-if --protocol tcp --match tcp --dport 53  --match mark --mark 0x3 --jump ACCEPT
```

Comment:
Did the necessary cleanup of marking nodes in inline has been done?

Note here that I don't know at all what I'm talking about when its question of inline.

Reply:
The goal is that go_dns is use even if you are reg or not and if the device is reg the the go_dns will proxy the request (like dns enforcement)
Btw we have to decide if we allow the dns traffic in the forward table (accept direct dns to 8.8.8.8 per example) or deny it.